libs/components/popovers/src/lib/modules/dropdown/dropdown.component.html

```diff
@@ -1,3 +1,18 @@
+<span
```

Comment:
Can we bump this to the bottom of the file since it's a lower priority than the main section of the template?